Release database emulator v4.1.0 (#1534)
firebase/firebase-tools: changelog.txt

* Allow the RTDB emulator to hot-reload rules file on changes.
* Allows emulated Cloud Functions to talk to production RTDB/Firestore if the emulators are not running.
* Fixes an issue where internal logs would sometimes appear in stdout.

> * Database emulator should accept all jwt algorithms and arbitrary gauth tokens

Reviewer: 1. make sure it's a sentence w/ a period at the end.
1. Is it a particular database (RTDB, Firestore) that accepts these?
1. JWT is fairly well known, what is `gauth`?

```suggestion
* Improves emulator support for JWT algorithms and arbitrary gauth tokens.
```
